ee/docker-compose.ch.yml

```diff
@@ -73,7 +73,6 @@ services:
         command: ./ee/bin/docker-ch-dev-web
         ports:
             - '8000:8000'
-            - '8234:8234'
```

Comment:
I think this is right, but tagging @fuziontech who was more expertise on this

Reply:
oh didn't mean to commit this, will remove